farao-dichotomy-api/src/main/java/com/farao_community/farao/dichotomy/api/DichotomyEngine.java

```diff
@@ -58,20 +59,34 @@ public DichotomyEngine(Index<T> index, IndexStrategy<T> indexStrategy, NetworkSh
      * Use this constructor to use the engine with the soft-interruption feature
      */
     public DichotomyEngine(Index<T> index, IndexStrategy<T> indexStrategy, InterruptionStrategy interruptionStrategy, NetworkShifter networkShifter, NetworkValidator<T> networkValidator, String runId) {
-        this(index, indexStrategy, interruptionStrategy, networkShifter, networkValidator, DEFAULT_MAX_ITERATION_NUMBER, runId);
+        this(index, indexStrategy, interruptionStrategy, networkShifter, networkValidator, null, DEFAULT_MAX_ITERATION_NUMBER, runId);
+    }
+
+    /**
+     * Use this constructor to use the engine with the soft-interruption feature and network-export
+     */
+    public DichotomyEngine(Index<T> index, IndexStrategy<T> indexStrategy, InterruptionStrategy interruptionStrategy, NetworkShifter networkShifter, NetworkValidator<T> networkValidator, NetworkExporter networkExporter, String runId) {
```

Comment:
Good point from our beloved rabbit: perhaps it would be better to introduce a builder instead of having 6 different constructors with slight variations in the parameters.